_pytest/config.py

```diff
@@ -1136,7 +1140,11 @@ def determine_setup(inifile, args):
                 if rootdir.join("setup.py").exists():
                     break
             else:
-                rootdir = ancestor
+                dirs = get_dirs_from_args(args)
```

Comment:
It will be used in other places as well - at least that's what I have locally at the moment.